src/main/java/com/jagrosh/jagtag/libraries/Functional.java

```diff
@@ -63,12 +63,36 @@
                     return in[1];
                 return in[2];
             }, "|then:", "|else:"),
-            
+
+            // performs a conditional and outputs true/false
+            new Method("eval", (env, in) -> {
```

Comment:
I feel like this isn't solving the issue.

Reply:
Why not? I think case fallthrough can be solved by the eval method, I just need one more function, which would be `{logic:&|{eval}|{eval}}`

Reply:
eval might be possible to exploit. Thats just why... you know?

Reply:
It's not an actual !eval from anywhere. It's the same evaluator that `{if}` uses. Maybe take a look at the code first? :)

Reply:
I actually don't think eval should be part of this PR; it doesn't seem related to switch statements, and I'd rather it be called something like `boolean` or `conditional`, and be added at the same time as `logic`

Reply:
That's a fair point. I'll move it to another PR perhaps.